pyKiwi-test.py

```diff
@@ -3,27 +3,36 @@
 from button import Button
 from size import Size
 from pos import Pos
+from text import Text
 
 import styles
 import orientation
 import colour
 import events
+import pygame
 
 def on_button_click():
     print 'button clicked'
 
 class TestGame(Game):
     def __init__(self):
         super(TestGame, self).__init__()
+        self.view = []
```

Comment:
so are we deciding to go with the list approach? What about positioning? Are we going to go with relative positioning?